<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editor</class>
 <widget class="QWidget" name="editor">
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="editorPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>652</width>
        <height>207</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="editorLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="controlsWidget" native="true">
         <layout class="QHBoxLayout" name="controlsLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="boldButton">
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="italicButton">
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="underlineButton">
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="justifyLeftButton"/>
          </item>
          <item>
           <widget class="QToolButton" name="justifyCenterButton"/>
          </item>
          <item>
           <widget class="QToolButton" name="justifyRightButton"/>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="backgroundButton">
            <property name="toolTip">
             <string>Draw Widget Background</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="colorButton">
            <property name="toolTip">
             <string>Font Color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KComboBox" name="fontSizeComboBox">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Font Size</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string notr="true">10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">48</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="fontFamilyComboBox">
            <property name="toolTip">
             <string>Font Family</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="AdjustableClock::ComponentWidget" name="componentWidget" native="true"/>
       </item>
       <item>
        <widget class="KTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="richTextTab">
          <attribute name="title">
           <string>Rich text</string>
          </attribute>
          <layout class="QVBoxLayout" name="richTextLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWebView" name="webView" native="true">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>100</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="webViewLine">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="sliderSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSlider" name="zoomSlider">
               <property name="toolTip">
                <string>Zoom: 100%</string>
               </property>
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sourceTab">
          <attribute name="title">
           <string>Source</string>
          </attribute>
          <layout class="QVBoxLayout" name="sourceLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="metaDataPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>635</width>
        <height>244</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Meta Data</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="identifierLabel">
         <property name="text">
          <string>Identifier:</string>
         </property>
         <property name="buddy">
          <cstring>identifierLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="identifierLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>nameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="nameLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description:</string>
         </property>
         <property name="buddy">
          <cstring>descriptionLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="descriptionLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="authorLabel">
         <property name="text">
          <string>Name of Authors:</string>
         </property>
         <property name="buddy">
          <cstring>authorLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="authorLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="emailLabel">
         <property name="text">
          <string>Emails of Authors:</string>
         </property>
         <property name="buddy">
          <cstring>emailLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KLineEdit" name="emailLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="websiteLabel">
         <property name="text">
          <string>Websites of Authors:</string>
         </property>
         <property name="buddy">
          <cstring>websiteLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KLineEdit" name="websiteLineEdit"/>
       </item>
       <item row="7" column="1">
        <widget class="KLineEdit" name="licenseLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="licenseLabel">
         <property name="text">
          <string>License:</string>
         </property>
         <property name="buddy">
          <cstring>licenseLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="KLineEdit" name="versionLineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="versionLabel">
         <property name="text">
          <string>Version:</string>
         </property>
         <property name="buddy">
          <cstring>versionLineEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>AdjustableClock::ComponentWidget</class>
   <extends>QWidget</extends>
   <header>ComponentWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
